simplq/src/components/pages/Admin/Token.jsx

```diff
@@ -6,29 +6,33 @@ import NotificationsOffIcon from '@material-ui/icons/NotificationsOffSharp';
 import CallIcon from '@material-ui/icons/Call';
 import moment from 'moment';
 import LoadingIndicator from 'components/common/LoadingIndicator';
-import { TokenRequestFactory } from 'api/requestFactory';
+import { notifyToken } from 'api/requestFactory';
+import { useDeleteToken } from 'store/asyncActions';
 import useRequest from 'api/useRequest';
+import { useDispatch } from 'react-redux';
 import styles from './admin.module.scss';
 
-function Token(props) {
-  const name = props.token.name;
-  const tokenId = props.token.tokenId;
-  const tokenNumber = props.token.tokenNumber;
-  const contactNumber = props.token.contactNumber;
-  const notifiable = props.token.notifiable;
-  const tokenCreationTimestamp = props.token.tokenCreationTimestamp;
+function Token({ token }) {
+  const name = token.name;
+  const tokenId = token.tokenId;
+  const tokenNumber = token.tokenNumber;
+  const contactNumber = token.contactNumber;
+  const notifiable = token.notifiable;
+  const tokenCreationTimestamp = token.tokenCreationTimestamp;
   const [notifying, setNotifying] = useState(false);
   const [isNotifyHovering, setIsNotifyHovering] = useState(false);
-  const [didNotify, setDidNotify] = useState(props.token.tokenStatus === 'NOTIFIED');
+  const [didNotify, setDidNotify] = useState(token.tokenStatus === 'NOTIFIED');
```

Comment:
When you dispatch action (e.g. `notifyToken`), you will have `actionStatus` in redux store. the status can be `pending`, `fulfilled` or ` rejected`. You can trigger actions based on that status. Check `JoinForm` for an example.